compiler/pipes/calc-bad-vars.cpp

```diff
@@ -547,6 +549,11 @@ class CalcBadVars {
     for (const auto &func : call_graph.functions) {
       if (into_interruptible[func]) {
         func->is_interruptible = true;
+        if (unlikely(func->class_id && func->class_id->is_required)) {
+          std::string func_name = func->name;
+          std::replace(func_name.begin(), func_name.end(), '$', ':');
+          kphp_error(false, fmt_format("{} cannot be interruptible", func_name).c_str());
```

Comment:
What is it for?

Reply:
It `offsetGet()` function calls interruptible function, this error will be printed. Without this, it would fail in `C++ to bin` stage with unclear error.